func.py

```diff
@@ -0,0 +1,5 @@
+def greet():
+    msg = "Hello, GitHub!"
```

Comment:
```suggestion
    greeting_message = "Hello, GitHub!"
```
Suggestion: Consider renaming `msg` to `greeting_message` for better clarity.